### ISSUE TITLE
Map pins / Circle Radius

### ISSUE THREAD

**Issue by sighjones** [TheBuro/Kanna#1]
is it possible to remove the blue pin, and just display the white pin box with the icon that relates to the spot, in place of the Kanna K icon - and when clicked it opens to the right to reveal the location name and info button? Also increase the radius of the circle to the size illustrated -  see attached

**Comment by MobilePie** [comment]
I'm not sure where to see the attachments, is there definitely one here?

If you provide the new pin icons as white box with icon inside, then we can use that. We can't control the callout opening to the right, it always appears above the pin.

**Comment by sighjones** [comment]
i dont think we should have ‘double’ pins (one coming out of the top of another..) 
- can we just have the white one with the icon inside and the name to the right?
- i’ll check with the client what they want to do..

s

sigh jones
sigh@the-buro.com
m: +44(0)7977 223 806

The Buro Ltd

F6 The Foundry
Beehive Yard
Bath
BA1 5BT

The Schoolhouse
Vineyards
Paragon
Bath 
BA1 5NA

t: +44(0)1225 331637

www.the-buro.com 
www.freakishkid.com

skype: theburo
twitter: theburo

On 25 Nov 2015, at 13:14, MobilePie notifications@github.com wrote:

I'm not sure where to see the attachments, is there definitely one here?

If you provide the new pin icons as white box with icon inside, then we can use that. We can't control the callout opening to the right, it always appears above the pin.

**Comment by MobilePie** [comment]
We can't have dynamic text on the pin, you can just supply an image. It's on the second pin (called callout) where you can add a title and subtitle

**Comment by SMKBuro** [comment]
ahh - ok.. could we do something like this then - i think its the same as it is now, except the blue marker pins are flat (they will be have icons in them), so they take up less room on the map.. attached.

sigh jones
sigh@the-buro.com
m: +44(0)7977 223 806

The Buro Ltd

F6 The Foundry
Beehive Yard
Bath
BA1 5BT

The Schoolhouse
Vineyards
Paragon
Bath 
BA1 5NA

t: +44(0)1225 331637

www.the-buro.com 
www.freakishkid.com

skype: theburo
twitter: theburo

On 25 Nov 2015, at 13:26, MobilePie notifications@github.com wrote:

We can't have dynamic text on the pin, you can just supply an image. It's on the second pin (called callout) where you can add a title and subtitle

**Comment by MobilePie** [comment]
I have added the new icons
